src/serializer.ts

```diff
@@ -105,9 +105,6 @@ export class Serializer {
             return new clazz();
         }
         let resultConstructor: new() => any = this.getClass(clazz, obj, discriminatorField);
```

Comment:
The type hint is required by our tslint.json file. See https://github.com/kaiu-io/serializer/blob/master/tslint.json#L49